zbus_macros/src/proxy.rs

```diff
@@ -744,9 +745,10 @@ fn gen_proxy_property(
             | PropertyEmitsChangedSignal::Const => {
                 let cached_getter = format_ident!("cached_{}", method_name);
                 let cached_doc = format!(
-                    " Get the cached value of the `{property_name}` property, or `None` if the property is not cached.",
+                    "Get the cached value of the `{property_name}` property, or `None` if the property is not cached.",
```

Comment:
same here.

Reply:
Revert in f511f8fd0a50a81b7682ff091ac6323642207fdc.